pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = ["hatch-vcs", "hatchling"]
+
+[project]
+name = "bump-dictionary"
+dynamic = ["version"]
+license = { file = "LICENSE" }
+authors = [{ name = "Neurobagel Developers" }]
+maintainers = [{ name = "Sebastian Urchs", email = "sebastian.urchs@mcgill.ca" }]
```

Comment:
This could probably use an update, huh? Or at least should include your name

Reply:
I can update this to include all our names. Should I still provide your email?